src/main/java/seedu/address/model/food/Carbohydrate.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.model.food;
+
+public class Carbohydrate extends Macronutrient {
+
+    public Carbohydrate(int amount) {
+        super(amount, 4);
+    }
+
+    @Override
+    public String toString() {
+        return "MacronutrientType: Carbohydrate\n"
+                + "Amount: "
+                + this.amount + "\n"
+                + "Caloric Count: " + this.totalCalories + "\n";
+    }
```

Comment:
Perhaps can refactor out a string format method in the  superclass (i.e. Macronutrient) that formats the toString() method found in the Protein, Carb, Fat subclasses so as to reduce code duplication.